qt/widgets/common/src/FindFilesWorker.cpp

```diff
@@ -122,6 +123,30 @@ void FindFilesWorker::run() {
   finishSearching(result);
 }
 
+/**
+ * Validate that the search text is not empty and contains no unsupported
+ * characters.
+ *
+ * @return a boolean indicating whether the search text is valid or not.
+ */
+bool FindFilesWorker::validateSearchText() {
+  std::string error;
+
+  if (m_parameters.searchText.empty()) {
+    error = "No files specified.";
+  } else if (!isASCII(m_parameters.searchText)) {
+    error = "An unsupported non-ASCII character was found in the search text.";
+  }
+
+  if (!error.empty()) {
+    std::vector<std::string> filenames;
+    finishSearching(createFindFilesSearchResult(
```

Comment:
Shuold we be doing the full search as part of the validation of the input?

Reply:
I don't think this is doing a full search, just creating a 'search result' containing an error message. This should be doing the same as was previously being done on line 61

Reply:
Ah sorry, my misunderstanding from reading the function name.